.github/workflows/checkForTranslations.yaml

```diff
@@ -0,0 +1,31 @@
+# https://crontab.guru/crontab.5.html
+
+name: Check for l10n updates
+
+on:
+
+  workflow_dispatch:
+
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '1 * * * *' # Once every hour (minute 1)
+
+jobs:
+  update-translations:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout main
+      uses: actions/checkout@v2
+
+    - name: Push changes
+      run: |
+        git config --global user.name github-actions
+        git config --global user.email github-actions@github.com
+        git remote add l10n https://github.com/nvdaaddons/addonName # Replace with the repo name, for example, clipContentsDesigner
```

Comment:
I'd be nice not to hardcode name of the repository. Have you tried playing with `GITHUB_REPOSITORY` to get the repo name?